scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -1508,6 +1508,7 @@ def _log_benchmark_run(self):
           'data_format': self.params.data_format,
           'rewrite_config': self.rewriter_config,
           'optimizer': self.params.optimizer,
+          'session_config': create_config_proto(self.params),
```

Comment:
Thanks for identifying the root cause! I can see [here](https://github.com/tensorflow/models/blob/master/official/utils/logs/logger.py#L391) in the code of model zoo it does list the device without using any session config, but I don't see how adding the session_config here can solve the problem, would you help to explain how?

Reply:
Oh I saw your other change to the model garden code, sorry I missed that.